pkg/controller/middleware/email_verified_test.go

```diff
@@ -169,7 +160,7 @@ func TestRequireEmailVerified(t *testing.T) {
 			w.Flush()
 
 			if got, want := w.Code, tc.code; got != want {
-				t.Errorf("expected %d to be %d", got, want)
```

Comment:
We use this format for test failures across the project and I'd prefer to keep it like this. Is there a strong reason to change it?

Reply:
The more or less google standard go is "got before want".  Besides this particular message (even if keeping the order the same) "expected 200 to be 400" is not the best message.

Reply:
It's still "got before want" in both the init and print. 

We use this same error output for most HTTP status tests throughout the entire project, and I think consistency is more important. 

While we also aren't bound by Google's internal recommendations, I don't think this is in violation.

Reply:
Kind of, but it reads weird. Anyway. While not bound it's still better to follow the general style.
🤷 — I was going to fix all of those one by one. Note that unit test errors are also expected to be uppercased in general — since they are whole sentences.